app/views/templates/test.html.slim

```diff
@@ -0,0 +1,42 @@
+.container-fluid
+  .page-header
+    h1 {{ test.name }}
+    h4 ng-if='!organization || !test' Loading...
+    h4 ng-if='organization && test' {{ organization | orgName }} > {{ test.project | projectName }}
+  .row
+    .col-lg-4.col-md-6.col-xs-12
+      .summary.panel.panel-default
+        .panel-heading.clearfix
+          .label.label-info.pull-right.apiId ng-if='test.key' tooltip='Test key' {{ test.key }}
+          h3.panel-title Details
+        .panel-body.test-details
+          p.loading ng-if='!test'
+            em Loading...
+          div ng-if='test'
+            .labels.clearfix
+              .label.label-primary
+                a href='#' ui-sref='org.projects.show({ projectName: test.project.name })' {{ test.project | projectName }}
+                |  {{ test.projectVersion }}
+              .label.label-info ng-if='test.category' {{ test.category }}
+              .label.label-default ng-repeat='tag in test.tags' {{ tag }}
+              .label.label-warning ng-repeat='ticket in test.tickets' {{ ticket }}
+            dl
+              dt
+                strong First run
+              dd {{ test.firstRunAt | amDateFormat: 'lll' }}
+              dt
+                strong Last run
+              dd {{ test.lastRunAt | amDateFormat: 'lll' }}
+              dt
+                strong Number of results
+              dd {{ test.resultsCount }}
+              dd
+                strong Last status
+              dt.test-details-status
+                span.glyphicon.glyphicon-thumbs-up.test-details-passed ng-if='test.passing && test.active' tooltip='Passed' tooltip-append-to-body='true'
+                span.glyphicon.glyphicon-thumbs-down.test-details-failed ng-if='!test.passing && test.active' tooltip='Failed' tooltip-append-to-body='true'
+                span.glyphicon.glyphicon-ban-circle.test-details-inactive ng-if='!test.active' tooltip='Inactive' tooltip-append-to-body='true'
+              dd
+                strong Contributors
+              dd
+                user-avatar ng-repeat='contribution in test.contributions' user='contribution.user' tooltip='{{ contribution.user.name }}' tooltip-append-to-body='true'
```

Comment:
In the current implementation, the list of contributors can be empty for some tests (and that is the case for some of the current tests). You should add a conditional `.empty` text to allow for the possibility.